### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by bicycle1885**
This updates the README.md page to describe more details we provide. I mimicked README of [JuMP.jl](https://github.com/JuliaOpt/JuMP.jl), because their page looks concise but appealing to new users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+    * Readers for FASTA, FASTQ and .2bit file formats
+* `Bio.Align`: Sequence alignment
+    * Alignment data structures
+    * Pairwise alignment algorithms
+* `Bio.Intervals`: Genomic intervals and annotations
+    * Genomic intervals with annotations
+    * Readers for BED and BigBed file formats
+* `Bio.Structure`: Molecular structures
+    * Macromolecular structures (protein)
+    * Reader for PDB file format
```
Comment (Ward9250) [inline README.md]: Can we add

```md
* `Bio.Var`: Biological Variation
    * Mutation counting
    * Genetic and evolutionary distances
```

**Comment by Ward9250** [comment]
LGTM! I only have one line comment.

File: README.md (hunk 1 above)
Comment (bicycle1885) [inline README.md]: Done 👍

**Comment by bicycle1885** [comment]
I think separating links and badges for the latest release and the development version is better as JuMP.jl does. But it can be done in a different pull request.

**Pull request merged**